docs/AWS_Integration_Guide/Concepts/AWS_Integration_concepts.md

```diff
@@ -0,0 +1,91 @@
+
+![Example Things](images/things.png "Example Things")
+
+!!! Note
+    Things are automatically added to AWS IoT Core upon completing the integration setup.
+
+### Thing types
+
+Thing types are containers that store configuration and other device-related information shared by all Things of the same type to simplify their bulk management. Within the integration, they are created automatically when a new device is added to AWS from Coiote DM and they are based on device **Manufacturer** and **Model name**. In case a new device with a specific **Manufacturer** and model name pair can be matched with an existing thing type, then it will be associated with it automatically.
+
+![Example Thing types](images/thingtypes.png "Example Thing types")
+
+!!! note
+    Note that you cannot modify a once created thing type, but you can deprecate (allowing no new devices to be associated with it) or delete it when there are no things associated with a given Thing type.
+
+## Device Shadows
+
+A device shadow is a structure that stores the device state and represents it in the form of a JSON file, making the device data available to applications and services regardless of device connection to Coiote DM. To synchronize device state information between the Coiote DM and AWS IoT Core, shadows feature the mechanism of *reported* and *desired* values.
+
+- **Reported values** section - presents the current device state as reported by the device itself (and mediated by Coiote DM) in a JSON file structure within a device shadow.
+- **Desired values** section - used for requesting changes in the reported section of the device [Operation shadow](#operation-shadow).
+
+![Integration Device Shadows](images/deviceshadows.png "Integration Device Shadows")
+
+For the purposes of the integration, a default of three different shadows is established for each connected thing: an unnamed shadow, a datamodel shadow and an operation shadow.
+
+### Classic Shadow
+
+The Classic shadow (also unnamed shadow) is used for storing connectivity parameters of a LwM2M device (such as registered lifetime, last lifetime refresh, queue mode, LwM2M URI, device registration status etc.). The `reported` state refreshes upon each change in these parameters that is reported by Coiote DM (**Register** or **Update** message from device).
+
+### Operation Shadow
+
+The Operation Shadow is where you request your LwM2M operations to be executed. To this end, the `desired` values of the device state are used. Thus, you can perform any LwM2M 1.0 operation on the device by defining it inside the `desired` values section. Also, to check if the operation execution was successful, the `reported` values section of the Operation Shadow is used (but only in case of the READ, WRITE and READ COMPOSITE operations).
+
+#### Communication flow
+
+A value change using the `desired` device state is formulated as the one below:
```

Comment:
Could we create a diagram here? Diagrams are always a better option than description.

Reply:
I added a diagram - please check it out